src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstallTask.groovy

```diff
@@ -0,0 +1,87 @@
+package wooga.gradle.paket.unity.tasks
+
+import groovy.transform.Internal
+import org.gradle.api.file.FileCollection
+import org.gradle.api.internal.ConventionTask
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.OutputDirectory
+import wooga.gradle.paket.base.utils.internal.PaketLock
+import wooga.gradle.paket.base.utils.internal.PaketUPMWrapperReference
+import wooga.gradle.paket.base.utils.internal.PaketUnityReferences
+
+import java.util.function.Function
+/**
+ * A task that installs packages onto an Unity Project
+ */
+abstract class PaketUnityInstallTask extends ConventionTask {
```

Comment:
Mislseading name, can be confused with PaketUnityInstall.

Reply:
Will make more explicit.